### PR TITLE
feat(topology): add specific details for all supported workloads

### PR DESCRIPTION
**Pull request by divyanshiGupta**
**Story:** https://github.com/janus-idp/backstage-plugins/issues/270

Added specific detail items for SS/DS/CJ/J resources as per OCP. Also added Jobs section in resources tab of CJ resource.

https://github.com/janus-idp/backstage-plugins/assets/20724543/a9750051-5e22-405e-905a-46ce01c56067

Screenshot 2023-05-17 at 3 57 58 PM

### CODE REVIEW

File: plugins/topology/src/components/Topology/TopologySideBar/TopologyCronJobDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <TopologyWorkloadDetails resource={resource}>
+      <TopologySideBarDetailsItem label="Schedule">
+        {resource.spec?.schedule}
+      </TopologySideBarDetailsItem>
+      <TopologySideBarDetailsItem label="Concurrency policy">
+        {resource.spec?.concurrencyPolicy}
+      </TopologySideBarDetailsItem>
+      <TopologySideBarDetailsItem label="Starting deadline seconds">
+        {resource.spec?.startingDeadlineSeconds
+          ? `${resource.spec.startingDeadlineSeconds} second`
```
Comment (invincibleJai) [inline plugins/topology/src/components/Topology/TopologySideBar/TopologyCronJobDetails.tsx:20]: should we handle plural as well i.e second or seconds?

File: plugins/topology/src/components/Topology/TopologySideBar/TopologyCronJobDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          : 'Not configured'}
+      </TopologySideBarDetailsItem>
+      <TopologySideBarDetailsItem label="Last schedule time">
+        <Timestamp
+          date={resource.status?.lastScheduleTime}
```
Comment (invincibleJai) [inline plugins/topology/src/components/Topology/TopologySideBar/TopologyCronJobDetails.tsx:25]: should we render Timestamp if lastScheduleTime is defined?

File: plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPanel.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.topology-job-pod-ring svg {
```
Comment (invincibleJai) [inline plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPanel.css:1]: I know we are not consistent with styles across plugins but we need to decide on how to handle it like scoping to plugin level i.e `bs-topology-*` . WDYT

cc @debsmita1

File: plugins/topology/src/components/Topology/TopologySideBar/TopologyCronJobDetails.tsx (hunk 1 above)
Comment (divyanshiGupta) [reply plugins/topology/src/components/Topology/TopologySideBar/TopologyCronJobDetails.tsx]: It is just `second` in ODC but handled it here.

**Comment by invincibleJai** [comment]
LGTM, verified the changes.

**Pull request merged**